ziti/pki/store/store.go

```diff
@@ -37,14 +37,15 @@ type Store interface {
 	// Returns an error if it failed to store the bundle.
 	Add(string, string, bool, []byte, []byte) error
 
-	// Chain concats an intermediate cert and a newly signed certificate bundle and adds the chained cert to the store.
+	// Chain concats an signing cert and a newly signed certificate bundle and adds the chained cert to the store.
 	//
 	// Args:
-	//  The intermediate CA name.
+	//  The signing CA name.
+	//  The destination CA name.
```

Comment:
what's a 'destination CA'?

Reply:
Under which CA directory the chain should be stored. For intermediates we should store the chain with the intermediate CA, but server certs are stored in the signing CA, so we should store it in the signing CA dir.